src/Illuminate/Support/Str.php

```diff
@@ -2119,4 +2119,15 @@ public static function flushCache()
         static::$camelCache = [];
         static::$studlyCache = [];
     }
+    @param  string  $email
```

Comment:
Your comment is broken:
```suggestion
   /*
    * @param  string  $email
```

Reply:
And you might need some space
```suggestion

    @param  string  $email
```